skillmap/src/components/makecodeFrame.tsx

```diff
@@ -166,7 +164,7 @@ class MakeCodeFrameImpl extends React.Component<MakeCodeFrameProps, MakeCodeFram
 
     protected onMessageReceived = (event: MessageEvent) => {
         const data = event.data as pxt.editor.EditorMessageRequest;
-        if (this.state.frameState === "opening-project") this.setState({ loadPercent: Math.min((this.state.loadPercent || 0) + 4, 95) });
+        if (this.state.frameState === "opening-project") this.setState({ loadPercent: Math.min((this.state.loadPercent || 0) + 7, 95) });
```

Comment:
This change is completely unnecessary and unrelated, but we never reach past ~half and for consecutive loads often not even a quarter  of the progress bar, and that felt bad to me :( with this we get 'close' to completion right around when it disappears and shows the editor.